pkg/v1/hash.go

```diff
@@ -19,12 +19,16 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	sha256simd "github.com/minio/sha256-simd"
 	"hash"
 	"io"
+	"os"
 	"strconv"
 	"strings"
 )
 
+const HasherSHA256 = "sha256"
```

Comment:
Maybe we should even have this be `var HasherSHA256 hash.Hash`, which we may override at `init` time based on the experiment env var? Then callers can use either `v1.Hasher("sha256")` or `v1.HasherSHA256` more or less interchangeably.